ui/settings.ts

```diff
@@ -19,6 +19,7 @@ export class Settings {
     balance_save_frequency = 24;
     last_balance_save = 0;
     include_crypto2crypto: boolean;
+    include_gas_costs: boolean;
```

Comment:
Can you instead of setting the type simply assign it to `true`, the default, here?